app/src/main/res/values-en/strings.xml

```diff
@@ -1202,4 +1202,5 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="achievement_lifesaver_solved_X">You solved %d quests that help emergency services and first responders!</string>
     <string name="quest_airConditioning_no_name_title">"Does this place (%s) have air conditioning?"</string>
     <string name="quest_airConditioning_name_type_title">"Does %1$s (%2$s) have air conditioning?"</string>
+    <string name="quest_seating_name_title">Does this place have seating?</string>
```

Comment:
Please revert this change, `values-en/strings.xml` will automatically be overwritten with the contents of `values/strings.xml`.